### PR TITLE
Added roslint. Implemented suggestions. Changed to raw pointers.

### PR DESCRIPTION
**Pull request by JWhitleyAStuff**
- Adds roslint as a dependency.
- Implements all suggested roslint changes.
- Changes passing around of shared_ptrs to raw pointers
- Changes all containers for CAN IDs to uint32_ts.

### CODE REVIEW

File: src/pacmod3_ros_msg_handler.cpp [hunk 1]
```diff
@@ -27,704 +30,738 @@ void LockedData::setData(std::vector<unsigned char> new_data)
   _data = new_data;
 }
 
-void Pacmod3TxRosMsgHandler::fillAndPublish(const int64_t& can_id,
-                                            std::string frame_id,
-                                            ros::Publisher& pub,
-                                            std::shared_ptr<Pacmod3TxMsg>& parser_class)
+void Pacmod3TxRosMsgHandler::fillAndPublish(
+const int64_t& can_id,
```
Comment (zoakesAStuff) [inline src/pacmod3_ros_msg_handler.cpp]: This probably ought to be indented one
```suggestion
    const int64_t& can_id,
```

**Comment by driscoll85** [comment]
Looks good, holding approval until tested by @rgunjala93

**Comment by zoakesAStuff** [comment]
Adding @rgunjala93 as a reviewer, so he'll get notified if there's further changes

**Comment by JWhitleyAStuff** [comment]
Removing PR. Found a better way than raw pointers.

**Pull request closed without merging**